content/6_Advanced/DP_More.mdx

```diff
@@ -105,16 +114,100 @@ Queue with 2 stacks is used to remove a factor of $\mathcal{O}(\log N)$ in [USAC
 
 ## DP on Broken Profile
 
-<Resources>
+<!-- <Resources>
 	<Resource
 		source="cp-algo"
 		title="DP on Broken Profile"
 		url="dynamic_programming/profile-dynamics.html"
 	>
 		{' '}
 	</Resource>
-</Resources>
+</Resources> -->
 
-<IncompleteSection />
+Broken profile DP is a subset of bitmask DP. Problems falling under this category generally have the following properties:
+1. They're about filling a 2D grid.
+2. One of the dimensions (usually rows) is much smaller than the other.
+3. When filling the grid, each cell depends only on adjacent cells.
+4. The cells don't have many possible values (usually only 2).
+
+The third property is especially important, as it means that we can process the cells column-by-column (imagine a snake wrapping around the grid). We then only need to care about the rightmost processed cell in each row (hence the name "broken profile").
+
+The fourth property suggests that we should use a bitmask to represent that broken profile.
+
+<Warning>
```

Comment:
Is this warning necessary?

Reply:
I just added it to show that there are different kinds of broken profiles, but since it's so niche, I'm fine with removing it